checkov/cloudformation/checks/resource/aws/EC2Credentials.py

```diff
@@ -1,10 +1,14 @@
+import base64
+import binascii
 from typing import List
+from base64 import b64decode
```

Comment:
`binascii` is an unused import. wither you can keep the `base64` one and remove `base64 import b64decode` or keep it and remove just the `base64`, but then you need to adjust the usage a few lines lower.